docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,22 +1,157 @@
+<p className="subtitle">This documentation helps you control what context from your codebase is used by Cody. You can do it via </p>
 
-This guide will walk you through the process of ignoring specific files or folders in your codebase when using Cody. 
+- Cody Context Filters (Cody Enterprise Only)
+- Cody Ignore File (All Cody users)
+
+## Context Filters
+
+<Callout type="note">Context Filters is available for all Cody Enterprise users.</Callout>
+
+Admins on the Sourcegraph Enterprise instance can use the Cody Context Filters to determine which repositories Cody can use as the context in its requests to third-party LLMs. This feature is only supported with the following requirements:
+
+- A valid Cody Enterprise license and a running Sourcegraph version `>=5.4.0`
+- Works if you're using Cody for VS Code, Cody for JetBrains, and Cody for Web as your client
+- Your Cody client meets the following version specs: VS Code `>=1.20.0` and JetBrains `>=6.0.0`
+- Setting the `cody-context-filters-enabled` feature flag to `true`
+
+Administrators can configure the `cody.contextFilters` field in the site configuration with the repos that they wish Cody to exclude or only include using the following structure:
+
+```
+{
+ // Optional. If defined, it must have at least one
+ // field set (either `include` or `exclude`).
+ 'cody.contextFilters': {
+ // Optional. If defined, it must have at least one item.
+ include: [
+ {
+ // Required. Should follow the RE2 syntax.
+ repoNamePattern: '^github\\.com\\/sourcegraph\\/.*'
+ }
+ ],
+ // Optional. If defined, it must have at least one item.
+ exclude: [
+ {
+ // Required. Should follow the RE2 syntax.
+ repoNamePattern: '.*analytics.*'
+ }
+ ]
+ }
+}
+```
+
+### How include and exclude rules work
+
+The `include` and `exclude` rules define the repositories Cody can use for context filtering to third-party LLMs. The rules can be defined in the following combination:
+
+1. `cody.contextFilters` field is not defined
+
+By default, there are no restrictions on the repositories Cody can use for context in requests to third-party LLMs. All repositories are included, and none are excluded.
+
+2. Only `include` rules are specified
+
+Cody is restricted from using content from repositories whose names match any specified patterns in the `include` field. Since no `exclude` rules are specified, no repositories are explicitly excluded. An `include` field might contain a catch-all pattern (e.g., a `regexp` matching any string). In this case, Cody can access content from any repository, as a single match in the `include` rules is sufficient for inclusion. For example,
+
+```
+{
+ 'cody.contextFilters': {
+ // Only repositories whose names either start with 'github.com/sourcegraph/'
+ // or contain 'cody' are allowed.
+ include: [
+ {repoNamePattern: '^github\\.com\\/sourcegraph\\/.+'},
+ {repoNamePattern: '.*cody.*'}
+ ]
+ }
+}
```

Comment:
@MaedahBatool It will be nice to get some color coding on the UI here, but not a blocker. Just a better UX